### PR TITLE
Better Claim Formatting

### PR DESCRIPTION
**Pull request by infinitebash**

### CODE REVIEW

**Review by lleyton — DISMISSED** [review]
Remove the crabs.

**Review by lleyton — APPROVED** [review]
Fine.

File: README.md [hunk 1]
```diff
@@ -1,33 +1,27 @@
...
-Day 16
-Day 17
-Day 18
-Day 19
-Day 20
-Day 21
-Day 22
-Day 23
-Day 24
-Day 25
```
Comment (iggyglass) [inline README.md]: ur mom gay lol.

**Review by iggyglass — COMMENTED** [review]
bruh

**Pull request merged**